src/components/Form.js

```diff
@@ -38,9 +40,9 @@ export default class Form extends Component {
     const edit = typeof props.formData !== "undefined";
     const liveValidate = props.liveValidate || this.props.liveValidate;
     const mustValidate = edit && !props.noValidate && liveValidate;
-    const {definitions} = schema;
+    const { definitions } = schema;
```

Comment:
Thanks for this! 

Couldn't we add it to eslint config to make sure it remains consistent with time?